src/Umbraco.Web/Editors/DashboardController.cs

```diff
@@ -61,6 +61,13 @@ public async Task<JObject> GetRemoteDashboardContent(string section, string base
             var version = UmbracoVersion.SemanticVersion.ToSemanticString();
             var isAdmin = user.IsAdmin();
 
+            if(!IsAllowedUrl(baseUrl))
+            {
+                var errorMsg = $"The following URL is not listed in the allowlist for ContentDashboardUrl in the Web.config: {baseUrl}";
+                Logger.Error<DashboardController>(errorMsg);
+                throw new HttpResponseException(Request.CreateErrorResponse(HttpStatusCode.BadRequest, errorMsg));
```

Comment:
I don't think we should include the full error message in the error response, only in the log entry, and instead just return something more generic like "Could not load dasboard"

Reply:
Makes sense 🙌 Will fix up